nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -221,6 +221,11 @@ in
       security.pam.services.sddm.enableKwallet = true;
       security.pam.services.slim.enableKwallet = true;
 
+      # Update the start menu for each user that has `isNormalUser` set.
+      system.activationScripts.plasmaSetup = stringAfter [ "users" "groups" ]
+        (concatStringsSep "\n"
+          (mapAttrsToList (name: value: "${pkgs.su}/bin/su ${name} -c kbuildsycoca5")
```

Comment:
Shouldn't this be a direct path via `${kdeWhatever}/bin/kbuildsycoca5`? Does this affect users that aren't running kde (which is possible through a custom ~/.xsession file)?

Reply:
> Does this affect users that aren't running kde (which is possible through a custom ~/.xsession file)?

This only applies if ```config.services.xserver.desktopManager.plasma5.enable``` is ```true``` so if somebody sets that and then overrides the environment using ```~/.xsession``` it would run even when KDE isn't active.